### DISCUSSION TITLE
Any "local build" or "dry run" ?

### DISCUSSION THREAD

**Discussion by sjeon87** [ObsidianPublisher/obsidian-github-publisher#256] [Q&A — Plugin]
I would like to see the expected result after pushing to GitHub repo (before actually pushing it). For example, to test updated configurations.

So I wonder if there is any way to do so.

**Comment by Mara-Li** [comment]
Sorry, there are none.
But, you can disable the auto merging branch, so you can check the branch created before manually merging it.

**Comment by sjeon87** [comment]
Thanks for quick response!
